MAUI/Badge-View/Badge-customization.md

```diff
@@ -298,3 +298,49 @@ Content = sfBadgeView;
 
 ![.NET MAUI Badge View Corner Radius](badge-customization_images/net_maui_badge_notifications_corner_radius.png)
 
+## Alignment of badge
+
+You can align the badge view using the Center, Start, and End properties of BadgeAlignment property.
```

Comment:
Align the badge view using the Center, Start, and End properties of BadgeAlignment property.